package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "social-platform-donut",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lakshyabatman/Social-Platform-Donut.git"
```

Comment:
?

Reply:
I initiated the Git on vue project.I removed it. :)